README.md

```diff
@@ -197,6 +197,8 @@ var DeviceInfo = require('react-native-device-info');
 | Total Memory               | `getTotalMemory()`                 | `number` e.g. 1995018240                                                                      | Total amount of memory on the device                                                                               |
 | Max Memory                 | `getMaxMemory()`                   | `number` e.g. 268435456                                                                       | ANDROID ONLY - see https://developer.android.com/reference/java/lang/Runtime.html#maxMemory()                       |
 | App Name                   | `getApplicationName()`             | `string` e.g.Learnium Mobile                                                                               |                                                                                                                  |
+| Total Disk Capacity        | `getTotalDiskCapacity()`           | `number` e.g. 68719476736                                                                     | Total storage in bytes                                                                                               |
+| Free Disk Storage          | `getFreeDiskStorage()`             | `number` e.g. 1995018240                                                                      | Free storage in bytes (Android - showing free external storage only)                                                       |
```

Comment:
Seems like you're using an outdated fork (I completely rewrote the README file two days ago)
Could you update your fork to latest HEAD?
Don't hesitate if you need some help!

Reply:
Updated! Thanks :)